### PR TITLE
Add cache limitation documentation

### PR DESCRIPTION
**Pull request by tung-vu-td**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,7 +99,9 @@ To add an event to local buffer, you can call `TreasureData`'s `addEvent` or `ad
 ```
 
 
-Specify the database and table to which you want to import the events. The total length of database and table must be shorter than 129 chars.
+Specify the database and table to which you want to import the events. The total length of database and table must be shorter than 129 chars. Each table will cache no more than 10000 events.
```
Comment (kietdo360) [inline README.md:102]: I think cache if from Keen managed storage, not specific for table within TD right ?

Comment (tung-vu-td) [reply README.md:102]: We map `database.table` to collection in Keen as Keen does not have notion of database/table. So the collection is indeed map 1 - 1 to the table. Writing the doc this way helps not to introduce the Keen's collection concept in the SDK documentation.

**Review by huylenq — APPROVED** [review]
LGTM

**Review by kietdo360 — APPROVED** [review]
LGTM

**Review by nukala — APPROVED** [review]
Reviewed for Data Exfiltrations

**Pull request merged**